custom-recipes/nlp-visualization-wordcloud/recipe.json

```
@@ -1,115 +1,129 @@
-            "type": "BOOLEAN",
-            "defaultValue": false,
-            "mandatory": true
-        },
-        {
-            "name": "separator_subcharts",
-            "label": "Subcharts",
-            "type": "SEPARATOR"
-        },
-        {
-            "name": "subchart_column",
-            "type": "COLUMN",
-            "columnRole": "input_dataset",
-            "allowedColumnTypes": [
-                "string"
-            ],
-            "label": "Categorical column",
-            "description": "Optional column to generate one word cloud per category",
-            "mandatory": false
-        }
-    ],
-    "resourceKeys": []
+    {
+      "name": "text_column",
+      "type": "COLUMN",
+      "columnRole": "input_dataset",
+      "allowedColumnTypes": ["string"],
+      "label": "Text column",
+      "mandatory": true
+    },
+    {
+      "name": "language",
+      "type": "SELECT",
+      "getChoicesFromPython": true,
+      "label": "Language",
+      "mandatory": true,
+      "defaultValue": "en"
+    },
+    {
+      "name": "language_column",
+      "label": "Language column",
+      "type": "COLUMN",
+      "description": "Column containing ISO 639-1 language codes",
+      "columnRole": "input_dataset",
+      "allowedColumnTypes": ["string"],
+      "visibilityCondition": "model.language == 'language_column'",
+      "mandatory": false
+    },
+    {
+      "name": "separator_text_handling",
+      "label": "Text simplification",
+      "type": "SEPARATOR"
+    },
+    {
+      "name": "remove_stopwords",
+      "label": "Clear stopwords",
+      "description": "Remove so-called 'stop words' (the, I, a, of, ...). This transformation is language-specific.",
+      "type": "BOOLEAN",
+      "defaultValue": true,
+      "mandatory": true
+    },
+    {
+      "name": "remove_punctuation",
+      "label": "Clear punctuation",
+      "description": "Remove punctuation characters e.g., ! ? ( ) . This transformation is language-specific.",
+      "type": "BOOLEAN",
+      "defaultValue": true,
+      "mandatory": true
+    },
+    {
+      "name": "case_insensitive",
+      "label": "Case-insensitive",
+      "description": "Normalize case to count 'You' and 'you' as the same word. The most common case will be displayed.",
+      "type": "BOOLEAN",
+      "defaultValue": false,
+      "mandatory": true
+    },
+    {
+      "name": "separator_display",
+      "label": "Display parameters",
+      "type": "SEPARATOR"
+    },
+    {
+      "type": "INT",
+      "name": "max_words",
+      "label": "Maximum number of words",
+      "minI": 1,
+      "defaultValue": 100
+    },
+    {
+      "type": "STRINGS",
+      "name": "color_list",
+      "label": "Color palette",
+      "description": "A list of colors to randomly color each word. Supported formats are X11 color names, hexadecimal color codes or matplotlib color shortcuts",
```

To make it fit on one line:
```suggestion
      "description": "List of colors to randomly color each word. Supported formats are hexadecimal color codes and matplotlib color names.",
```